tkltest/generate/unit/augment.py

```diff
@@ -488,3 +488,8 @@ def __print_test_counter(counter):
     sys.stdout.write('\r')
     sys.stdout.write('* {}'.format(counter))
     sys.stdout.flush()
+
+def save_div(a,b):
```

Comment:
@haim-kermany: should the function name be `safe_div`?

Reply:
fixed